src/images/kafka.rs

```diff
@@ -0,0 +1,124 @@
+use crate::{core::WaitFor, Image};
+use std::{collections::HashMap, path::PathBuf};
+
+const CONTAINER_IDENTIFIER: &str = "confluentinc/cp-kafka";
+const DEFAULT_TAG: &str = "6.1.1";
+
+pub const KAFKA_PORT: u16 = 9093;
+const ZOOKEEPER_PORT: u16 = 2181;
+
+#[derive(Clone, Debug)]
+pub struct KafkaArgs {
+    pub bash_pathname: PathBuf,
+    pub options: Vec<String>,
+    pub commands: String,
+}
+
+impl Default for KafkaArgs {
+    fn default() -> Self {
+        let starter_commands = format!(
+            "echo 'clientPort={}' > zookeeper.properties; \
+echo 'dataDir=/var/lib/zookeeper/data' >> zookeeper.properties; \
+echo 'dataLogDir=/var/lib/zookeeper/log' >> zookeeper.properties; \
+zookeeper-server-start zookeeper.properties & \
+export KAFKA_ZOOKEEPER_CONNECT='localhost:{}'; \
+export KAFKA_ADVERTISED_LISTENERS='PLAINTEXT://localhost:{},BROKER://localhost:9092'; \
+. /etc/confluent/docker/bash-config && \
+/etc/confluent/docker/configure && \
+/etc/confluent/docker/launch",
+            ZOOKEEPER_PORT, ZOOKEEPER_PORT, KAFKA_PORT
+        );
+
+        Self {
+            bash_pathname: PathBuf::from("/bin/bash"),
+            options: vec!["-c".to_owned()],
+            commands: starter_commands,
+        }
+    }
+}
+
+impl IntoIterator for KafkaArgs {
+    type Item = String;
+    type IntoIter = ::std::vec::IntoIter<String>;
+
+    fn into_iter(self) -> Self::IntoIter {
+        let mut args = vec![self.bash_pathname.into_os_string().into_string().unwrap()];
+        args.extend(self.options.into_iter());
+        args.push(self.commands);
+        args.into_iter()
+    }
+}
+
+#[derive(Debug)]
+pub struct Kafka {
+    arguments: KafkaArgs,
+    env_vars: HashMap<String, String>,
+    tag: String,
+}
+
+impl Kafka {
+    pub fn with_env_vars(mut self, env_vars: HashMap<String, String>) -> Self {
+        self.env_vars.extend(env_vars.into_iter());
+        self
+    }
```

Comment:
Do we ever need to modify any environment variables? If not, this can go away.

Reply:
Addressed in 81be19a.